app/src/main/java/com/sopt/now/Login/LoginActivity.kt

```diff
@@ -52,17 +45,38 @@ class LoginActivity : AppCompatActivity() {
 
     private fun initLoginBtnClickListener() {
         binding.btnLoginSignin.setOnClickListener {
-            if (binding.etLoginId.text.toString() == user?.id && binding.etLoginPw.text.toString() == pw) {
-                // 로그인 성공
-                Toast.makeText(this, R.string.login_success, Toast.LENGTH_SHORT).show()
-                Intent(this, HomeActivity::class.java).apply {
-                    putExtra(USER, user)
-                    startActivity(this)
+            val loginRequest = getLoginRequestDto(binding.etLoginId.text.toString(), binding.etLoginPw.text.toString())
+            authService.login(loginRequest).enqueue(object : Callback<ResponseSignupDto> {
```

Comment:
```suggestion
            val loginRequest = RequestLoginDto(
                authenticationId = binding.etLoginId.text.toString(),
                password = binding.etLoginPw.text.toString()
            )
            authService.login(loginRequest).enqueue(object : Callback<ResponseSignupDto> {
```

는 어떨까요? 하단의 getLoginRequestDto 메서드가 필요해보이진 않는 것 같아요!

Reply:
아하!! 감사합니다😊